full-flow.sh

```diff
@@ -12,6 +17,25 @@ DB2_PATH=$(sed -n 's/^SYNCHRONIZER_SQLITE_PATH="\([^"]*\)"/\1/p' .env)
 rm $DB1_PATH
 rm $DB2_PATH
 
+echo -e "build go binary"
+git clone https://github.com/0xPARC/pod2-onchain.git tmp/pod2-onchain
+cd tmp/pod2-onchain
+go build
+mv ./pod2-onchain ../../pod2-onchain
+cd -
+
+# if the sample pod proof does not exist, create it
+if [ ! -d "tmp/podproof" ]; then
+	echo -e "generate a first pod proof to have a sample for the Groth16 verifier"
+	cargo test --release -p common gen_sample_pod_proof -- --nocapture
```

Comment:
The test `gen_sample_pod_proof` is defined as ignored, so this line will not run the test.

Reply:
Moreover if I try to really run this test at this step, it fails because the `pk, vk, r1cs` are not found (the trusted setup hasn't been executed yet)